unittests/CMakeLists.txt

```diff
@@ -108,7 +114,7 @@ else ()
     target_link_libraries(rbus_gtest.bin rbus gtest gcov)
 endif()
 
-install (TARGETS rbus_event_server rbus_test_server rbus_gtest.bin
+install (TARGETS rbus_event_server rbus_test_server rbus_gtest.bin sample_rbus_session_mgr
```

Comment:
the install target for ```sample_rbus_session_mgr``` must be within the compile time macro.. ie it must within line 80 above..